docs/modules/automation-testing/pages/start-a-mixed-session.adoc

```diff
@@ -13,20 +19,58 @@ image:automation-testing:launch-device-context.png[width=800,alt="A context of l
 
-image:automation-testing:start-second-session-context.png[width=1000,alt="A context to start a second automation session after manual session"]
+=== If the first session is manual
+
+Run an Appium or Basic Appium 2 automation script on the device as a manual session is running, using the same user. Once the automation script starts running, a banner on the manual session appears with a message  _Automation Script is running on this device_. Now you can view your script running on the device and fix issues in realtime.
+
+image:automation-testing:manual-as-first-session.png[width=1000,alt="A context to start a second automation session after a manual session"]
+
+=== If the first session is automation
+
+Make sure you are logged in as the same user that started the automation session.
+
+Follow one of the following possible methods to start a manual session in an ongoing automation session:
+
+* From the *Device list*, select the *Launch* button while the device was running automation to enter the mixed session:
+
++
+
+image:automation-testing:launch-mixed-session-device-list.png[width=1000,alt="A utilizing device with a Launch button to start a mixed session"]
+
+* Open the *Session Overview* page of the session, then select the *Session Viewer* button:
+
++
+
+image:automation-testing:launched-mixed-session-session-overview.png[width=1000,alt="The Session Viewer button in Session Overview of an ongoing automation session"]
+
+* In the session list, highlight the ongoing automation session (by clicking the session name) then choose *Session Viewer* on the action menu:
+
++
+
+image:automation-testing:launched-mixed-session-session-list-hightlight.png[width=1000,alt="The Session Viewer option when highlighting an ongoing automation session in session list"]
+
+* Alternatively, in the session list, right-click the session, and select *Session Viewer*:
+
++
+
+image:automation-testing:launch-mixed-session-session-list-right-click.png[width=1000,alt="The Session Viewer option when right-clicking an ongoing automation session in session list"]
 
 Now you can view your script running on the device and fix issues in realtime.
 
 == Limitations/Notes
 
-* In a Mixed session, both the Manual and Automation session must be initiated by the same user.
+* In a mixed session, both the Manual and Automation session must be initiated by the same user.
 
 * Mixed session are only available for Kobiton 4+ devices.
 
-* Mixed sessions can only be initiated with Appium automation tests; other automation frameworks, e.g. Espresso, XCUITest, are not currently supported.
+* Mixed sessions can only be initiated with Appium or Basic Appium 2 automation tests; other automation frameworks, e.g. Espresso, XCUITest, are not currently supported.
+
+* Manual actions in a mixed session is not yet available when a Basic Appium 2 script is running.
```

Comment:
Performing manual actions in a mixed session is not supported for Basic Appium 2 script execution.